app/src/main/res/layout/revision_card_fragment.xml

```diff
@@ -65,7 +66,7 @@
           android:clipToPadding="true"
           android:onClick="@{() -> viewModel.onPreviousCardClicked()}"
           android:visibility="@{viewModel.subtopicId == 1 ? View.GONE : View.VISIBLE}"
-          app:cardBackgroundColor="@color/color_def_white"
+          app:cardBackgroundColor="@color/component_color_shared_screen_primary_background_color"
```

Comment:
This is incorrect, in both cases the background should be white.

Suggest creating a shared background so that you can use it in `Revision Card` tab items too.

Something like 

`component_color_shared_revision_card_background_color`

Reply:
Apply this change for `next` card too and also in tablet mock.

Reply:
Done.